FreeSurfer/FreeSurferPipeline-v5.3.0-HCP.sh

```diff
@@ -75,6 +76,26 @@ else
 fi
 log_Msg "seed_cmd_appendix: ${seed_cmd_appendix}"
 
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+ProcessingMode=`opts_GetOpt1 "--processing-mode" $@`
+ProcessingMode=`opts_DefaultOpt $ProcessingMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
+
+# -- T2w image
+
+if [ "${T2wInputImages}" = "NONE" ]; then
+  ComplianceMsg+=" --t2=NONE"
+  Compliance="LegacyStyleData"
+fi
+
+check_mode_compliance "${ProcessingMoce}" "${Compliance}" "${ComplianceMsg}"
```

Comment:
Typo.

Reply:
Ahh ... where did this come from ... corrected.